script.js

```diff
@@ -8,15 +8,15 @@ window.addEventListener('resize', () => {
   }
 });
 
-function openNav() {
-  document.getElementById('mySidebar').classList.add('sidebar');
-  document.getElementById('mySidebar').classList.remove('menu-lists');
-}
+// function openNav() {
+// document.getElementById('mySidebar').classList.add('sidebar');
+// document.getElementById('mySidebar').classList.remove('menu-lists');
+// }
 
-function closeNav() {
-  document.getElementById('mySidebar').classList.remove('sidebar');
-  document.getElementById('mySidebar').classList.add('menu-lists');
-}
+// function closeNav() {
+// document.getElementById('mySidebar').classList.remove('sidebar');
+// document.getElementById('mySidebar').classList.add('menu-lists');
+// }
```

Comment:
- [ ] Great job so far @AliShahSafdari and @berylcl your contact form is well validated. However, to adhere to [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean. If you need to review a previous version of your code, you can always use git history. :+1: